x-pack/plugins/osquery/public/plugin.ts

```diff
@@ -95,6 +95,7 @@ export class OsqueryPlugin implements Plugin<OsqueryPluginSetup, OsqueryPluginSt
         storage: this.storage,
         kibanaVersion: this.kibanaVersion,
       }),
+      isOsqueryAvailable: useIsOsqueryAvailableSimple,
```

Comment:
do we need `Simple` in the name?

Reply:
This is to show, that it has less functionality than useIsOsqueryAvailable (the one with React-query). Imho, makes sense, do you have any suggestions :)

Reply:
just keep the `Simple`? :)

Reply:
`useIsQueryAvailableNonIO`? I see what Patryk is saying. Maybe something that says NonIO? I'm fine either way, but just a suggestion